### PR TITLE
RAND: Fix code style

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
Just a few remaining style nit's in rand code.

##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Comment by FdaSilvaYY** [comment]
Ping @mspncp

File: crypto/rand/drbg_lib.c [hunk 1]
```diff
@@ -378,7 +378,7 @@ int RAND_DRBG_instantiate(RAND_DRBG *drbg,
         entropylen = drbg->get_entropy(drbg, &entropy, min_entropy,
                                        min_entropylen, max_entropylen, 0);
     if (entropylen < min_entropylen
-        || entropylen > max_entropylen) {
+            || entropylen > max_entropylen) {
```
Comment (mspncp) [inline crypto/rand/drbg_lib.c]: Is this extra indent optional or mandatory? And was it added by the indent script or by you? It's not that I dislike it (on the contrary), but I recall being told some time ago that the `||` should align after the opening brace, just like in argument lists (e.g. line 379). Also the Emacs indentation style is configured like that.

**Comment by mspncp** [comment]
@levitte I can merge it (and cherry-pick it to 1.1.1) this evening but feel free to do it yourself if you prefer.

File: crypto/rand/drbg_lib.c (hunk 1 above)
Comment (levitte) [reply crypto/rand/drbg_lib.c]: This is optional, so I think it's an ok change to make, even though we don't *have* to.

As for automatic indentation, I don't know of any that supports this extra indentation.  That's my primary reason for not making it mandatory (I wouldn't want to fight Emacs *every damn time*).
If someone finds an indent program that does support this kind of construct, we're all ears!

**Comment by levitte** [comment]
Merged.

master:
c2e33a05b1eb9dda988aebdeaa529973d7c22142 crypto/rand: fix some style nit's

1.1.1:
7f0e220f4d842a7694ed0f8ff9675745977ce9a0 crypto/rand: fix some style nit's

**Comment by thaij** [comment]
@levitte  I found a indent program in my centos distribution, and i looked up its man page. in the section 'BREAKING LONG LINE S', GNU indent breaks long lines before the boolean operators &&
       and || by combining option '-nbbo',  '-hnl' and '--line-length60', which is relevant with the extra indent in your context. Hopefully, this would help you.

**Pull request closed without merging**